src/routes/docs/__layout.svelte

```diff
@@ -193,26 +204,28 @@
 				{/if}
 			</div>
 		</div>
-		<div class="page-inner markdown-body">
-			<header>
-				<span>
-					{$page.path.split("/").join(" / ").substring(2)}
-					{$page.path === "/docs" ? " / overview" : ""}
-				</span>
-				<div class="header-right">
-					<HyperlinkButton
-						href="https://github.com/{links.github.owner}/{links.github
-							.siteRepo}/edit/main/src/routes/docs{currentPage.path ||
-							'/index'}.svx"
-						rel="noreferrer noopener"
-						target="_blank"
-					>
-						Edit this page
-					</HyperlinkButton>
-				</div>
-			</header>
-			<slot />
-		</div>
+		<PageTransition refresh={pagePath}>
```

Comment:
Why is a wrapper element necessary here? Is there no way to put the transition on the `markdown-body` div?